entity-framework/core/get-started/install/index.md

```diff
@@ -88,28 +89,28 @@ Although you can also use the `dotnet ef` commands from the Package Manager Cons
 
 The `dotnet ef` commands are included in current versions of the .NET Core SDK, but to enable the commands on a specific project, you have to install the `Microsoft.EntityFrameworkCore.Design` package:
 
-``` Console	
-dotnet add package Microsoft.EntityFrameworkCore.Design	
-```	
+``` Console
+dotnet add package Microsoft.EntityFrameworkCore.Design
+```
 
 For ASP.NET Core apps, this package is included automatically.
 
-> [!IMPORTANT]  	
+> [!IMPORTANT]
```

Comment:
Huh? Seriously? Why would it render any differently??

Reply:
Two trailing spaces tells some markdown parses to actually honor the newline. Otherwise all the newline is just treated like any other whitespace in HTML and the paragraph wraps as needed when rendering

Reply:
Yeah, I can see that in the VS Code preview. No super strong feelings, but should we let that non-standard behavior dictate it for us? Besides, @smitpatel's IDE will strip those trailing whitespaces anyway over time :)

(just to be sure there's no impact in the generated site: [preview](https://review.docs.microsoft.com/en-us/ef/core/get-started/install/index?branch=pr-en-us-1801#get-the-net-core-cli-tools) [live](https://docs.microsoft.com/en-us/ef/core/get-started/install/index#get-the-net-core-cli-tools))